man/coap.txt.in

```diff
@@ -39,15 +39,25 @@ examples directory.
 SEE ALSO
 --------
 *coap_attribute*(3), *coap_context*(3), *coap_encryption*(3), *coap_handler*(3),
-*coap_logging*(3), *coap_observe*(3), *coap_pdu_setup*(3), *coap_recovery*(3),
-*coap_resource*(3), *coap_session*(3) and *coap_tls_library*(3)
+*coap_keepalive*(3), *coap_logging*(3), *coap_observe*(3), *coap_pdu_setup*(3),
+*coap_recovery*(3), *coap_resource*(3), *coap_session*(3)
+and *coap_tls_library*(3)
 
 For example executables, see *coap-client*(5), *coap-rd*(5) and *coap-server*(5)
 
 FURTHER INFORMATION
 -------------------
-See "RFC7252: The Constrained Application Protocol (CoAP)" for further
-information.
+See
+
+"RFC7252: The Constrained Application Protocol (CoAP)"
+
+"RFC7641: Observing Resources in the Constrained Application Protocol (CoAP)"
+
+"RFC7959: Block-Wise Transfers in the Constrained Application Protocol (CoAP)"
+
+"RFC8323: CoAP (Constrained Application Protocol) over TCP, TLS, and WebSockets"
+
```

Comment:
I would add RFCs [7967](https://tools.ietf.org/html/rfc7967) and [8132](https://tools.ietf.org/html/rfc8132) here as well.

Reply:
Fixed